docs/integrations/databases/convex.mdx

```diff
@@ -45,87 +73,142 @@ The next step is to configure Convex with the issuer domain provided by Clerk. F
   alt="The 'Create new template' page of the JWT Templates page in the Clerk Dashboard. There is a red box surrounding the 'Issuer' section."
 />
 
-In your terminal, run the following command from your Convex app directory:
+In your `convex` folder, add an `auth.config.js` file with the following configuration:
+
+```ts title="convex/auth.config.js"
+export default {
+  providers: [
+    {
+      domain: "https://your-issuer-url.clerk.accounts.dev/",
+      applicationID: "convex",
+    },
+  ]
+};
+```
```

Comment:
This isn't rendering correctly, probably because `title` instead of `filename`
```suggestion
```ts filename="convex/auth.config.js"
export default {
  providers: [
    {
      domain: "https://your-issuer-url.clerk.accounts.dev/",
      applicationID: "convex",
    },
  ]
};
```
```

Reply:
great catch!!